src/ComponentMap.js

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
It seems like this file is very similar to [this model](https://github.com/Adyen/adyen-web/blob/master/packages/lib/src/core/ProcessResponse/PaymentMethodsResponse/PaymentMethodsResponse.ts)  that we have in the Web repository. Creating proper models/classes for the Objects is a good idea, as it improves readability and encapsulates all the logic in one place.

This 'model' PaymentMethodsResponse could potentially be applied here as far as I see!

Reply:
It is inspired by `PaymentMethodsResponse.ts`, yes :)
The only issue - there is no TypeScript yet applied to RN repo. 
I'll add this as improvement point for future!

Reply:
That is not an issue at all :-) Typescript helps to define types and make the syntax more strict and type-safe, but you can still create Models without TS by just using plain Javascript. You are at least able to encapsulate all the logic in that model. Later on, in case we migrate to TS, the refactoring will easier.

Reply:
Will try in next PR, thanks :)